src/assets/translations/it/labels.js

```diff
@@ -2,40 +2,64 @@ export const labels = {
   address: "Indirizzo",
   amount: "Importo",
   availables: "Disponibili",
+  bonded: "Vincolati",
   commission: "Commissione",
   commissionRate: "Tasso commissione",
+  cumulativeShare: "% Quota cumulata",
+  date: "Data",
   ddoReference: "Riferimento DDO",
   delegated: "Delegati",
+  delegator: "Delegante",
   delegatorAddress: "Delegante",
   depositEndTime: "Termine deposito",
   description: "Descrizione",
+  details: "Dettagli",
   did: "DID",
+  fee: "Onorario",
```

Comment:
Riusciamo a cambiare la traduzione in "Fee"? Non credo che "Onorario" sia la traduzione migliore